readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -105,7 +105,7 @@ def append_conf(self, **kwargs):
             'builder': "mkdocs",
             'docroot': docs_dir,
             'source_suffix': ".md",
-            'api_host': getattr(settings, 'SLUMBER_API_HOST', 'https://readthedocs.org'),
+            'api_host': getattr(settings, 'PRODUCTION_DOMAIN', 'https://readthedocs.org'),
```

Comment:
It's still useful, especially if we ever decide to use api.readthedocs.org or whatever. Splitting up these domain settings is important though, perhaps even in the future, have a specify domain setting for the API domain here.